packages/react-select/src/components/Select/useSelectStyles.ts

```diff
@@ -1,7 +1,16 @@
+import { SlotClassNames } from '@fluentui/react-utilities';
 import { makeStyles, mergeClasses } from '@griffel/react';
-import type { SelectState } from './Select.types';
+import type { SelectSlots, SelectState } from './Select.types';
 
+/**
+ * @deprecated
+ */
 export const selectClassName = 'fui-Select';
```

Comment:
For `Select` you can actually remove this class name instead of deprecating it since the component is not being exported from `@fluentui/react-components` yet:

```suggestion
```